database-mongo/index.js

```diff
@@ -35,8 +34,10 @@ const populate = (restaurant) => {
     });
```

Comment:
So I assume you just got rid of the find function on this page. Will you move the model and schema to its own file and call this populate and any find functions on a different page?